### DISCUSSION TITLE
vscode: `WebviewViews.onDidChangeVisibility`

### DISCUSSION THREAD

**Discussion by vince-fugnitto** [eclipse-theia/theia#11665] [Q&A] [answered]
@msujew I had a question regarding `WebviewViews.onDidChangeVisibility` which was introduced as part of https://github.com/eclipse-theia/theia/pull/10705, was there a specific reason we wanted an `Event<boolean>` and not `Event<void>` which the VS Code API declares?

https://github.com/eclipse-theia/theia/blob/2289366b82f6d013fd4de7b990b21d9037a74c32/packages/plugin/src/theia.d.ts#L4370

I checked both `1.53.2` and `main` and they both define it as `Event<void>` (and of course this will mark the API as `partial` in the compatibility report).

**Comment by msujew** [comment, marked as answer]
@vince-fugnitto Seems to be an oversight on my part when copying code from https://github.com/eclipse-theia/theia/pull/10137. It should probably be `Event<void>`.

**Reply by vince-fugnitto** [reply]
@msujew thanks! I'll make the update as part of my cleanup :)
